tools/topology/topology2/cavs/CMakeLists.txt

```diff
@@ -25,6 +25,9 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt.bin"
 
 # CAVS SDW topology with passthrough pipelines
 "cavs-sdw\;cavs-sdw"
+# CAVS SDW + DMIC topology with passthrough pipelines
+"cavs-sdw\;cavs-mtl-sdw-dmic\;NUM_DMICS=4,\
```

Comment:
why 4 mics?

Also we have to stop using passthrough pipelines, this isn't what we need for MTL where the multi-capture stream capability is needed. @ranj063 @juimonen FYI